plugin/src/main/scala/scoverage/CoverageFilter.scala

```diff
@@ -56,7 +61,7 @@ class RegexCoverageFilter(
 
   override def isFileIncluded(file: SourceFile): Boolean = {
     def isFileMatch(file: SourceFile) = excludedFilePatterns.exists(
-      _.matcher(file.path.replace(".scala", "")).matches
+      _.matcher(file.path).matches
```

Comment:
This creates a weird user experience where excluding `.*\/directory\/Class\.scala` will NOT match. Fixing it to make it easier to use.

Reply:
And does this still work the same for users that have been using this without include `.scala`?

Reply:
No. That would be a breaking change.

We could support both.

Or we leave it like is and "just" update the README to document more/better how this works.

Thoughts?

Reply:
Yea, since we just released 2.0.0 I want to avoid a breaking change here. If somehow we can support both, then great let's do that and make sure it's tested. If not, then let's document the current behavior and make it clear how it works.

Reply:
Agreed. Rolled back. Restored the current behaviour. Added a note to the README.